src/lib/states.js

```diff
@@ -50,12 +50,12 @@ class States {
           return hook[method](task)
             .then(info => { return info; })
             .catch(err => {
-              errors.push(new Error(`Error calling '${method}' for ${hook.featureName} : ${err.message}`));
+              errors.push(new Error(`Error calling '${method}' for ${hook.featureName} : ${err}`));
```

Comment:
Nice... generally what I do is:
`err.toString()` or `err ? err.message || err : 'unknown'`

Or maybe `(err || 'unknown').toString()` or just `err || 'unknown'`